create_instance_wNetwork.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
```

Comment:
Usage of same file naming rules is a good practice so doesn't mix different namings in your project. Rename it to `create_instance_with_network.sh`